scripts/apidoc_full.sh

```diff
@@ -22,7 +28,13 @@ fi
 CONFIG_BAK=${CONFIG_H}.bak
 cp -p $CONFIG_H $CONFIG_BAK
 
-scripts/config.py realfull
-make apidoc
-
-mv $CONFIG_BAK $CONFIG_H
+if in_mbedtls_repo; then
+    scripts/config.py realfull
+    make apidoc
+    mv $CONFIG_BAK $CONFIG_H
+elif in_tf_psa_crypto_repo; then
+    cd $OUT_OF_SOURCE_DIR
```

Comment:
```suggestion
    cd $OUT_OF_SOURCE_DIR
```
This is specific to all.sh and we want this script to work on its own. Here you can just build in tree:
`cmake -DCMAKE_BUILD_TYPE:String=Check .`.